### PR TITLE
update Artifacts for 2.0.0

### PR DESCRIPTION
**Pull request by lsampras**
- update artifacts for 2.0.0
- update podspec to remove payments and add location
- update git lfs to stop tracking payments and start tracking location
- update documentation

### CODE REVIEW

File: Feature Guides/integration-guide.md [hunk 1]
```diff
@@ -23,116 +22,117 @@ You also need to have [git-lfs](https://git-lfs.github.com/) installed on your m
...
-HaptikLib requires some permissions and custom properties to function.
-Add the following snippets in your `info.plist` file -
+HaptikLib requires some permissions and custom properties to function properly. Add the following snippets in your `info.plist` file.
 
-1. Enter the following exception domains for HaptikLib to work properly
+<details open>
+   <summary> Haptik only supports <b>Portrait</b> orientation </summary>
 
-   ```
-   <key>NSAppTransportSecurity</key>
```
Comment (tejas-haptik) [inline Feature Guides/integration-guide.md]: Why did we remove this? Not needed anymore?

File: Feature Guides/integration-guide.md [hunk 2]
```diff
@@ -263,136 +274,68 @@ B. **OTP Authentication**
...
-   	else {
-   	    NSLog(@"User Signup Failed!");
-   	}
+            else {
+               NSLog(@"User Signup Failed!");
+            }
    }];
    ```
 
-B. **Synchronous (with Customisable Loading Screen)**
```
Comment (tejas-haptik) [inline Feature Guides/integration-guide.md:342]: We removed this?

Comment (lsampras) [reply Feature Guides/integration-guide.md:342]: we removed the loading screen along with inbox

**Review by haptikrajasashtikar — APPROVED** [review]

File: HaptikLib.podspec [hunk 3]
```diff
@@ -42,12 +42,11 @@ Pod::Spec.new do |s|
...
-  # Haptik Payment Module Subspec
+  # Haptik Location Module Subspec
 
-  s.subspec 'Payments' do |payments|
-    payments.ios.deployment_target = '9.0'
-    payments.dependency    'PayUIndia-Custom-Browser', '5.8.1'
-    payments.vendored_frameworks = "Frameworks/Payments/HaptikPayments.framework"
+  s.subspec 'Location' do |location|
+    location.ios.deployment_target = '10.0'
+    location.vendored_frameworks = "Frameworks/Location/HaptikLocation.framework"
```
Comment (iamsimranjot) [inline HaptikLib.podspec:49]: Don't we need to weakly link the frameworks for the `CoreLocation` here?

Comment (lsampras) [reply HaptikLib.podspec:49]: @iamsimranjot 
`CoreLocation`/`MapKit`  are auto-linked so we don't need to specify them.
similar to `CoreData`.
we can add these as a preventive measure tho if needed

Comment (iamsimranjot) [reply HaptikLib.podspec:49]: Cool

**Review by iamsimranjot — APPROVED** [review]

**Pull request merged**